flake8_quotes/__init__.py

```diff
@@ -44,19 +44,27 @@ def __init__(self, tree, filename='(none)', builtins=None):
     def add_options(cls, parser):
         parser.add_option('--quotes', default='\'', action='store',
                           help='Quote to expect in all files (default: \')')
-        parser.add_option('--multiline-quotes', default='\'', action='store',
-                          help='Multiline quote to expect in all files (default: \')')
+        parser.add_option('--multiline-quotes', default='', action='store',
+                          help='Multiline quote to expect in all files (default: any)')
```

Comment:
As a user, "default: any" would make me assume `--multiline-quotes any` would work =/ Maybe we should say `...in all files (e.g. """, ''')`